UNRELEASED.md

```diff
@@ -20,6 +20,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 ### Development workflow
 
 - Removed the need to upload assets with each release ([#2346](https://github.com/Shopify/polaris-react/pull/2346))
+- Made running yarn splash along with the storybook optional ([#2372](https://github.com/Shopify/polaris-react/pull/2372))
```

Comment:
We could save people a click by telling them a bit more about this.

Here's a suggestion:

```suggestion
- Made it possible to prevent [“splash”](https://github.com/Shopify/polaris-react/tree/master/scripts/splash) from automatically running alongside `yarn dev`: `DISABLE_SPLASH=1 yarn dev` ([#2372](https://github.com/Shopify/polaris-react/pull/2372))
```